src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2282,6 +2282,9 @@ No\ metadata\ found.\ Creating\ empty\ entry\ with\ file\ link=No metadata found
 Processing\ file\ %0=Processing file %0
 Export\ selected=Export selected
 
+UnCombine\ merged\ citations=UnCombine merged citations
```

Comment:
Uncombine seems to be a proper english word. So the inword capitialization is not needed. But maybe `separate` instead of uncombine and unmerge would be better?

Reply:
Merge citations | Unmerge citations
Join citations | Separate citations

I would like to make it clear  that the two buttons correspond,
hence came `Unmerge`.

To `Separate` I associate `Join` as the opposite.
(English is not my primary language)

Which pair (of the above, or something else) would you prefer?

Reply:
I'm not familiar enough with the feature, but I would say join/separate are a bit clearer / non-technical terms. Maybe @Siedlerchr  has a preference.

Reply:
It does a bit more than just putting them in a pair of shared parentheses:
The entries under the same reference mark are sorted according to [comparatorForMulticite](https://github.com/antalk2/jabref/blob/cbd674b9785ca486721ec78ffeac53c1c5fffc30/src/main/java/org/jabref/gui/openoffice/OOBibBase.java#L632)
And there is a comment at [getCitationMarker](https://github.com/antalk2/jabref/blob/cbd674b9785ca486721ec78ffeac53c1c5fffc30/src/main/java/org/jabref/logic/openoffice/OOBibStyle.java#L492) saying:
// // E.g. (Olsen, 2005a, b) should be output instead of (Olsen, 2005a; Olsen, 2005b).

Reply:
I've got no strong opinion, but think separate/merge is also suited